src/ngraph/op/experimental/batch_dot.hpp

```diff
@@ -22,27 +22,35 @@ namespace ngraph
 {
     namespace op
     {
+        /// \brief Dot product for a batch of Rank 2 tensors.
         class BatchDot : public Op
         {
         public:
-            BatchDot(std::shared_ptr<Node> a,
-                     std::shared_ptr<Node> b,
-                     bool transpose_a,
-                     bool transpose_b);
+            /// \brief Constructs a batch dot product operation.
+            ///
+            /// \param arg0 The node producing the first argument.
+            /// \param arg1 The node producing the second argument.
+            /// \param transpose_0 Apply transpose to arg0.
+            /// \param transpose_1 Apply transpose to arg1.
```

Comment:
Can we add a more mathematical definition? Or at the very least, the axis order convention.

Reply:
Sure, I'll add these. Thanks.

Reply:
Fixed.